base/deprecated.jl

```diff
@@ -1269,11 +1269,13 @@ end
 # (2) base/linalg/qr.jl
 # (3) base/linalg/lq.jl
 
-@deprecate find(x::Number)            find(!iszero, x)
-@deprecate findnext(A, v, i::Integer) findnext(equalto(v), A, i)
-@deprecate findfirst(A, v)            findfirst(equalto(v), A)
-@deprecate findprev(A, v, i::Integer) findprev(equalto(v), A, i)
-@deprecate findlast(A, v)             findlast(equalto(v), A)
+@deprecate find(x::Number)             find(!iszero, x)
+@deprecate findnext(A, v, i)           findnext(equalto(v), A, i)
+@deprecate findfirst(A, v)             findfirst(equalto(v), A)
+@deprecate findprev(A, v, i)           findprev(equalto(v), A, i)
+@deprecate findlast(A, v)              findlast(equalto(v), A)
+@deprecate findnext(A::BitArray, v, i) findnext(equalto(v), A, i)
```

Comment:
Are these two deprecations really needed, given that we have more general versions above?

Reply:
Probably not, though I added them to ensure (mostly) the same specificity we have now. Sort of like your argument above about not removing `::Integer` from `start`.